main.tf

```diff
@@ -183,3 +186,72 @@ resource "null_resource" "confirm_ibm_mq_operator_operational" {
     }
   }
 }
+
+##############################################################################
+# Queue Manager
+##############################################################################
+
+resource "kubernetes_namespace" "ibm_mq_queue_manager_namespace" {
+  count = var.create_ibm_mq_queue_manager_namespace == true ? 1 : 0
+
+  metadata {
+    name = var.ibm_mq_queue_manager_namespace
+  }
+
+  timeouts {
+    delete = "30m"
+  }
+
+  lifecycle {
+    ignore_changes = [
+      metadata[0].annotations,
+      metadata[0].labels
+    ]
+  }
+}
+
+resource "helm_release" "ibm_mq_queue_manager" {
+  depends_on = [time_sleep.wait_catalog[0], helm_release.ibm_mq_operator, kubernetes_namespace.ibm_mq_queue_manager_namespace[0]]
```

Comment:
remove `time_sleep.wait_catalog[0]` as already present in `helm_release.ibm_mq_operator`